edgecontext.thrift

```diff
@@ -7,6 +7,16 @@ namespace java com.reddit.edgecontext
 */
 typedef string AuthenticationToken
 
+/** ISO 639-1 language identifier
+
+*/
+typedef string LanguageCode
+
+/** ISO 3166-1 region identifier
+
+*/
+typedef string RegionCode
+
 /** A two-character ISO 3166-1 country code
```

Comment:
As noted here we already have an ISO 3166-1 country code for the client coming on the edge context. I totally get that the two things may sometimes coincide but aren't always going to be the same since this is generally going to come from GeoIP type stuff so would indicate the current location of the client rather than the desired regional variant of a language but it would be good to document this so people are clear when to use which.

Reply:
Great call out. Added some extra description here to make the different use cases clearer.